src/hearings/resolvers/priority/priority.resolve.ts

```diff
@@ -38,8 +39,8 @@ export class PriorityResolver implements Resolve<RefDataModel[]> {
     );
   }
 
-  public getReferenceData$(serviceId): Observable<RefDataModel[]> {
-    return this.hearingsDataService.getRefData(HearingCategory.Priority, serviceId).pipe(
+  public getReferenceData$(serviceId, category: HearingCategory = HearingCategory.Priority): Observable<RefDataModel[]> {
```

Comment:
doesn't quite make sense to set a default value here, it should be passed by individual route config.

Reply:
This is there so it does not break priority

Reply:
So I will remove this and force the priority to have data category passed ok